### PR TITLE
feat: update velero to v1.13

### PR DESCRIPTION
**Pull request by Ani1357**
This MR updated velero to version 1.13(current latest).
Tested on Azure AKS cluster:
- Restored backups taken from velero 1.12 ✅
- Created backups for team containing multiple workloads ✅
- Backup/Restored workloads using different storage class backed up PVCs ✅

### CODE REVIEW

File: values/velero/velero.gotmpl [hunk 1]
```diff
@@ -19,9 +19,11 @@ metrics:
 snapshotsEnabled: false
 {{- end }}
 
-upgradeCRDs: false
+upgradeCRDs: true
```
Comment (j-zimnowoda) [inline values/velero/velero.gotmpl:22]: Do we need this?

File: values/velero/velero.gotmpl [hunk 2]
```diff
@@ -67,8 +69,10 @@ podSecurityContext:
   runAsUser: 1000
 
 configuration:
-  {{- if eq $sp.type "azureBlob" }}
+  uploaderType: restic
```
Comment (j-zimnowoda) [inline values/velero/velero.gotmpl:72]: Does it mean rustic will be always used?
If so do we need update in otomi.io?

File: values/velero/velero.gotmpl (hunk 1 above)
Comment (Ani1357) [reply values/velero/velero.gotmpl:22]: I tested it and on this version I did not encounter any issues. I'd prefer to keep this when/if it works

File: values/velero/velero.gotmpl (hunk 2 above)
Comment (Ani1357) [reply values/velero/velero.gotmpl:72]: Not exactly. In the most velero versions there is an alternative to restic(kopia). With this option we choose one over the other. Enabling the [Node Agent](https://velero.io/docs/main/file-system-backup/#install-velero-node-agent)(aka. File System Backup) is done via the `deployNodeAgent` parameter. What we can do maybe in a later stage is to accommodate this choice in otomi(restic or kopia).

File: values/velero/velero.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply values/velero/velero.gotmpl:22]: I am a bit concerned that for old Velero version the upgrade CRDs suddenly stopped working. If the upgradeCRDs only performs the `kubectl` command then we could do it via the upgrade scripts. I leave this decision to you.

File: values/velero/velero.gotmpl (hunk 2 above)
Comment (j-zimnowoda) [reply values/velero/velero.gotmpl:72]: ah now, I get it. Could you please add this info as a comment in code?

File: values/velero/velero.gotmpl (hunk 1 above)
Comment (Ani1357) [reply values/velero/velero.gotmpl]: Updated to use the kubectl installation

File: values/velero/velero.gotmpl (hunk 2 above)
Comment (Ani1357) [reply values/velero/velero.gotmpl:73]: Done

**Review by j-zimnowoda — APPROVED** [review]

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**